src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/tasks/CleanupEnvelopesDlqTask.java

```diff
@@ -48,12 +48,11 @@ public void deleteMessagesInEnvelopesDlq() throws ServiceBusException, Interrupt
             IMessage message = messageReceiver.receive();
             while (message != null) {
                 if (canBeDeleted(message)) {
-                    logMessage(message);
-                    messageReceiver.complete(message.getLockToken());
+                    deleteMessage(messageReceiver, message);
                 }
                 message = messageReceiver.receive();
             }
-
+            log.info("Finished processing messages in envelopes Dead letter queue.");
```

Comment:
This is not what this PR is about and it can be done later, but it would be nice to have the number of deleted messages in the log.

Reply:
@lgonczar removed the log message. I'll add it in a separate PR and will include the number of messages deleted